http/api_handler.go

```diff
@@ -119,8 +119,8 @@ func NewAPIHandler(b *APIBackend) *APIHandler {
 
 	sourceBackend := NewSourceBackend(b)
 	sourceBackend.SourceService = authorizer.NewSourceService(b.SourceService)
-	sourceBackend.NewBucketService = b.NewBucketService
-	sourceBackend.NewQueryService = b.NewQueryService
+	sourceBackend.LabelService = authorizer.NewLabelService(b.LabelService)
```

Comment:
I don't think it's necessary to wrap labelService in authorizer here. If the user has access to a bucket, we can assume that they can see the bucket's labels. You can move this line to have it be set in `NewSourceBackend` and remove the authorizer.

Reply:
thanks @jademcgough, fixed